### ISSUE TITLE
precompilation error

### ISSUE THREAD

**Issue by jlchan** [chriselrod/StrideArrays.jl#30]
Running into the following precompilation error
```julia
julia> using StrideArrays
[ Info: Precompiling StrideArrays [d1fa6d79-ef01-42a6-86c9-f7c551f8593b]
ERROR: LoadError: UndefVarError: maybestaticsize not defined
Stacktrace:
 [1] include
   @ ./Base.jl:386 [inlined]
 [2] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt64}}, source::Nothing)
   @ Base ./loading.jl:1213
 [3] top-level scope
   @ none:1
 [4] eval
   @ ./boot.jl:360 [inlined]
 [5] eval(x::Expr)
   @ Base.MainInclude ./client.jl:446
 [6] top-level scope
   @ none:1
in expression starting at /Users/jessechan/.julia/packages/StrideArrays/J1EsX/src/StrideArrays.jl:1
ERROR: Failed to precompile StrideArrays [d1fa6d79-ef01-42a6-86c9-f7c551f8593b] to /Users/jessechan/.julia/compiled/v1.6/StrideArrays/jl_NHJpCg.
Stacktrace:
 [1] error(s::String)
   @ Base ./error.jl:33
 [2] compilecache(pkg::Base.PkgId, path::String, internal_stderr::Base.TTY, internal_stdout::Base.TTY)
   @ Base ./loading.jl:1360
 [3] compilecache(pkg::Base.PkgId, path::String)
   @ Base ./loading.jl:1306
 [4] _require(pkg::Base.PkgId)
   @ Base ./loading.jl:1021
 [5] require(uuidkey::Base.PkgId)
   @ Base ./loading.jl:914
 [6] require(into::Module, mod::Symbol)
   @ Base ./loading.jl:901
```

**Comment by jlchan** [comment]
Ah, looks like it's the same as https://github.com/JuliaSIMD/LoopVectorization.jl/issues/285.

**Comment by chriselrod** [comment]
Thanks for the issue.
It should be fixed on master if you're on Julia >= 1.6.

I'll make a new release here after an LV release that'll let this work with older Octavian versions that're 1.5 compatible.

**Comment by jlchan** [comment]
Thanks for the quick fix!
